website_form_recaptcha/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016-2017 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+
+<odoo noupdate="1">
+    <record id="recaptcha_key_site" model="ir.config_parameter">
+        <field name="key">recaptcha.key.site</field>
+        <field name="value">Obtain from Google</field>
+    </record>
+    <record id="recaptcha_key_secret" model="ir.config_parameter">
+        <field name="key">recaptcha.key.secret</field>
+        <field name="value">Obtain from Google</field>
```

Comment:
Remove this file. Having wrong configurations is wrong, in general. Also, isn't it supposed that these keys are used per website? Websites have fields for such. Do not store things in 2 places, it's confusing.

Reply:
it has been done to:

* support server env for global conf on all websites
* support per-website override

I agree on removing fake values

Reply:
I have double-checked this and in v12, now that multiwebsite is mainstream, all settings for a website are always stored in the website object itself, and configurable by the config view.

I think the all-websites setting makes no sense anymore. The new approach makes more sense, and is UI-friendly. We'd have less code to maintain, and the workaround would be as simple as copy-paste from one website to the other, and a simple website admin user can do that without asking anybody.

Let's remove this completely please @cristinamartinrod